app/Http/Controllers/ImportController.php

```diff
@@ -144,7 +145,7 @@ public function postComplete(Request $request, Import $import) {
                     'tag_id' => $row['tag_id'],
                     'happened_on' => $row['happened_on'],
                     'description' => $row['description'],
-                    'amount' => (int) ($amount * 100)
+                    'amount' => Helper::rawNumberToInteger($amount)
```

Comment:
The ```rawNumberToInteger``` expects it to be a float, but the $amount seems to not be validated anywhere.

Reply:
You mean, we should do that on line 120, with a "numeric" rule?

Reply:
Oh, I did not see that one. I think you can also validate on arrays like so; something.*
All things in the "something" array must match the rule in that case.

Reply:
Alright, thanks!